webhooks/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+cat > .ksp-cakn << END_CONFIG
```

Comment:
Probable typo in `.ksp-cakn` (a and k swapped)

Reply:
**whistles quietly**